src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="txt-area">0</div>
+        <div className="input-area">
+          <div className="row">
+            <div>Ac</div>
+            <div>+/-</div>
+            <div>%</div>
+            <div className="orange">÷</div>
+          </div>
+          <div className="row">
+            <div>7</div>
+            <div>8</div>
+            <div>9</div>
+            <div className="orange">x</div>
+          </div>
+          <div className="row">
+            <div>4</div>
+            <div>5</div>
+            <div>6</div>
+            <div className="orange">-</div>
+          </div>
+          <div className="row">
+            <div>1</div>
+            <div>2</div>
+            <div>3</div>
+            <div className="orange">+</div>
+          </div>
+          <div className="row">
+            <div className="btn-zero">0</div>
+            <div>.</div>
+            <div className="orange">=</div>
+          </div>
+        </div>
+      </div>
+    );
```

Comment:
- [ ] Your calculator should match the design. Please stick with the design
- [ ] Please make improvements on the screen of the calculator
### Original design
![ssssssssss](https://user-images.githubusercontent.com/81727988/188870996-74ad7d4a-b913-411e-8f0f-b55e34a1206d.png)
### Your design
![your](https://user-images.githubusercontent.com/81727988/188871047-07cf3454-329b-4bf4-a768-73bd1c532289.png)